### DISCUSSION TITLE
Telegram Notifications

### DISCUSSION THREAD

**Discussion by aarondvail** [jellyfin/jellyfin#8190] [General]
I would like notifications via Telegram. I use it with the "arr" applications, a number of Arduino sketches,  OpenHAB, and Plex (Through OpenHAB).

PushBullet is a similar option, but PushBullet has a cap on number of messages a month, which I've used up in as quickly as a week.

**Comment by crobibero** [comment]
The webhook plugin should be able to do what you need.

https://github.com/jellyfin/jellyfin-plugin-webhook

**Reply by aarondvail** [reply]
I've been looking at it... and it's a bit complicated.

Found https://github.com/MindFlavor/Jellyfin.Plugins.Telegram

but it looks like it needs to be updated to 10.8
